build.gradle

```diff
@@ -23,11 +23,16 @@ repositories {
     mavenCentral()
     maven { url "https://hub.spigotmc.org/nexus/content/repositories/snapshots/" }
     maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
+    maven {
+        name 'velocity'
+        url 'https://repo.velocitypowered.com/snapshots/'
+    }
 }
 
 dependencies {
     compileOnly 'org.spigotmc:spigot-api:1.13-R0.1-SNAPSHOT'
-
+    compileOnly 'org.projectlombok:lombok:1.18.8'
```

Comment:
Excuse my lack of knowledge with Velocity, but does it require @Inject on fields? Can it not be used without?

Reply:
Yes, it needs this to Inject the dependencies from Velocity itself